src/main/java/org/ibp/api/java/genotype/SampleGenotypeService.java

```diff
@@ -0,0 +1,23 @@
+package org.ibp.api.java.genotype;
+
+import org.generationcp.middleware.domain.etl.MeasurementVariable;
+import org.generationcp.middleware.domain.genotype.SampleGenotypeDTO;
+import org.generationcp.middleware.domain.genotype.SampleGenotypeImportRequestDto;
+import org.generationcp.middleware.domain.genotype.SampleGenotypeSearchRequestDTO;
+import org.springframework.data.domain.Pageable;
+
+import java.util.List;
+
+public interface SampleGenotypeService {
+
+	List<Integer> importSampleGenotypes(String programUUID, Integer studyId,
+		List<SampleGenotypeImportRequestDto> genotypeImportRequestDtos);
+
+	List<SampleGenotypeDTO> searchSampleGenotypes(SampleGenotypeSearchRequestDTO searchRequestDTO, Pageable pageable);
+
+	long countSampleGenotypes(SampleGenotypeSearchRequestDTO searchRequestDTO);
```

Comment:
same comment that I've made in middleware with the names for countSampleGenotypes and countFilteredSampleGenotypes